### ISSUE TITLE
Pick attributes without docstrings

### ISSUE THREAD

**Issue by pawamoy** [pawamoy/mkdocstrings#13]
Just like we still select objects with no docstrings, we could also pick attributes without docstrings.

The user can then configure the plugin to render them or not.

**Comment by cs01** [comment]
Would an example of this be if a type annotation is present for the return type but there is no docstring for it (`Returns: ...`), the generated mkdocs would still have a `Returns` section?

**Comment by pawamoy** [comment]
No, I was talking about picking module variables, class variables and instance variables (in init methods) that have no docstrings, to be consistent with the selection method for other objects (we currently select methods/classes that have no docstrings as well, and rendering them or not based on configuration).

But what you wrote would indeed be an interesting feature. Sometimes the method/function is self-explanatory about what it returns, so printing its return type without having to add `Returns:` would be nice.
